x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/action/TransportStartDataFrameAnalyticsAction.java

```diff
@@ -218,12 +219,13 @@ private void estimateMemoryUsageAndUpdateMemoryTracker(StartContext startContext
                 // Validate that model memory limit is sufficient to run the analysis
```

Comment:
Let's also update the comment here to say we're just warning when limit is lower than estimate